source/slang/slang-compiler.h

```diff
@@ -1073,15 +1071,10 @@ namespace Slang
         List<SearchDirectory>   searchDirectories;
     };
 
-    /// Create a blob that will retain (a copy of) raw data.
-    ///
-    ComPtr<ISlangBlob> createRawBlob(void const* data, size_t size);
-
-
         /// Given a target returns the required downstream compiler
     PassThroughMode getDownstreamCompilerRequiredForTarget(CodeGenTarget target);
 
-    PassThroughMode getPassThroughModeForCPPCompiler(CPPCompiler::CompilerType type);
+    PassThroughMode getPassThroughModeForCPPCompiler(DownstreamCompiler::CompilerType type);
```

Comment:
Is this function still specific to C/C++, or should it be `...ForDownstreamCompiler`?